packages/react-components/source/react/library/icon/icons.js

```diff
@@ -198,6 +198,26 @@ const icons = {
     ),
   },
 
+  'chevron-double-down': {
+    medium: (
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M10.2929 3.29289L8 5.58579L5.70711 3.29289C5.31658 2.90237 4.68342 2.90237 4.29289 3.29289C3.90237 3.68342 3.90237 4.31658 4.29289 4.70711L7.29289 7.70711C7.68342 8.09763 8.31658 8.09763 8.70711 7.70711L11.7071 4.70711C12.0976 4.31658 12.0976 3.68342 11.7071 3.29289C11.3166 2.90237 10.6834 2.90237 10.2929 3.29289ZM10.2929 8.29289L8 10.5858L5.70711 8.29289C5.31658 7.90237 4.68342 7.90237 4.29289 8.29289C3.90237 8.68342 3.90237 9.31658 4.29289 9.70711L7.29289 12.7071C7.68342 13.0976 8.31658 13.0976 8.70711 12.7071L11.7071 9.70711C12.0976 9.31658 12.0976 8.68342 11.7071 8.29289C11.3166 7.90237 10.6834 7.90237 10.2929 8.29289Z"
```

Comment:
Have you run this through svgo to reduce its size? Take a look at the readme for the Icon component:

https://github.com/puppetlabs/design-system/blob/development/packages/react-components/source/react/library/icon/Icon.md#adding-icons-to-the-design-system